src/Tools/dotnet-monitor/Egress/AzureBlob/AzureBlobEgressPostConfigureOptions.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Tools.Monitor.Egress.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.Egress.AzureBlob
+{
+    /// <summary>
+    /// Fills AccountKey and SharedAccessSignature from Egress:Properties if they do not have values.
+    /// </summary>
+    internal sealed class AzureBlobEgressPostConfigureOptions :
+        IPostConfigureOptions<AzureBlobEgressProviderOptions>
+    {
+        private readonly ILogger<AzureBlobEgressPostConfigureOptions> _logger;
```

Comment:
Can you validate that this fixes https://github.com/dotnet/dotnet-monitor/issues/444

Reply:
This does not repro with the changes in this PR. I'll mark that this issue is fixed by this PR.